conductor_api/src/config.rs

```diff
@@ -114,13 +114,11 @@ pub fn default_persistence_dir() -> PathBuf {
     holochain_common::paths::config_root().join("conductor")
 }
 
-/// There might be different kinds of loggers in the future.
-/// Currently there is a "debug" and "simple" logger.
-/// TODO: make this an enum
+/// This is a config helper structure used to interface with the holochain logging subcrate.
```

Comment:
Hm, I actually thought we discussed providing some more info about how to use/configure logging as this is part of the user-facing reference documentation..

Reply:
Oh ok no problem! That's good thinking actually, thanks !